### PR TITLE
Useful information in README

### LINKED ISSUE #3: Missing README.md content
README.md is currently empty, content needs to be added

### PR DESCRIPTION
**Pull request by idan1356**
README file consists of:
- project description section
- license
- team members
fixes #3

### CODE REVIEW

**Review by Omeramsc — CHANGES_REQUESTED** [review]
I like the work with issues, keep it up! 
The title of the PR should not be that complicated, I think adding the logo can be considered as part of what been added to the README. "Useful information in README" (or something along those lines) is a fine name

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+
+##### Additional important Virtual environment commands:
+
+- You can enter the virtual machine by running `vagrant ssh`
+- Use `Vagrant halt` to shut the environment down
+- When finished, run `vagrant destroy -f` to tear down the environment
+
+### Contributing
+
+Please see our CONTRIBUTING.md file for details on how to contribute.
```
[the remark this answers is not in the document]
Comment (natifridman) [reply README.md:36]: You can always create another PR to add it to the README

**Review by Omeramsc — APPROVED** [review]
LGTM :)

**Comment by natifridman** [comment]
Waiting for 2 students' reviews

**Review by shirazyomtov — APPROVED** [review]
LGTM

**Review by natifridman — APPROVED** [review]
LGTM

**Pull request merged**